modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -73,7 +77,7 @@ The following table lists the approximate requests averaging 10 KB that can be s
 |===
 
 [NOTE]
-The internal load balancer runs on the controller VMs of Runtime Fabric. Size the VMs appropriately based on the amount and type of inbound traffic. You can only allocate half of the available CPU cores on each VM to the internal load balancer.
+The internal load balancer runs on the controller VMs of Runtime Fabric. Size the VMs appropriately based on the amount and type of inbound traffic. You can allocate only half of the available CPU cores on each VM to the internal load balancer.
```

Comment:
Omit "appropriately," because we follow it by specifying the appropriate measure.

Good catch on "only"!